rust-src/concordium_base/src/transactions.rs

```diff
@@ -182,11 +182,8 @@ pub enum TransactionType {
     ConfigureBaker,
     ///  Configure an account's stake delegation.
     ConfigureDelegation,
-    /// Token holder transaction. Introduced in Concordium protocol version 9.
-    TokenHolder,
-    /// Token governance transaction. Introduced in Concordium protocol version
-    /// 9.
-    TokenGovernance,
+    /// Token update transaction. Introduced in Concordium protocol version 9.
+    TokenUpdate,
```

Comment:
There are a couple of types changed in PR 674 and we need to insure that the JSON serialization has been propagated to the rust part of the code. I would suggest to add test cases to haskell/rust to confirm we are still aligned on that.